liboptv/include/segmentation.h

```diff
@@ -0,0 +1,47 @@
+/*  Objects related to calibration: exterior and interior parameters, glass
+    parameters. A Calibration object to rule them all and find them. 
+
+    References:
+    [1] Th. Dracos (editor). Three-Dimensional Velocity and Vorticity Measuring
+        and Image Analysis Techniques. Kluwer Academic Publishers. 1996. 
+        pp.165-168
+*/
+
+#ifndef SEGMENTATION_H
+#define SEGMENTATION_H
+
+#include "tracking_frame_buf.h"
+#include "parameters.h" 
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
```

Comment:
when I remove <string.h> I receive the warning about `memcpy`

`/Users/alex/Documents/openptv/alexlib/liboptv/src/segmentation.c:59:3: warning: 
      implicitly declaring library function 'memcpy' with type 'void *(void *,
      const void *, unsigned long)'
  memcpy(img0, img, imx*imy);
  ^
/Users/alex/Documents/openptv/alexlib/liboptv/src/segmentation.c:59:3: note: 
      include the header <string.h> or explicitly provide a declaration for
      'memcpy'
1 warning generated.`